src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.cs

```diff
@@ -1383,9 +1383,6 @@ public partial class DelegatingConfigHost : System.Configuration.Internal.IInter
         public virtual string GetConfigPathFromLocationSubPath(string configPath, string locationSubPath) { throw null; }
         public virtual System.Type GetConfigType(string typeName, bool throwOnError) { throw null; }
         public virtual string GetConfigTypeName(System.Type t) { throw null; }
-#pragma warning disable SYSLIB0003
-        public virtual void GetRestrictedPermissions(System.Configuration.Internal.IInternalConfigRecord configRecord, out System.Security.PermissionSet permissionSet, out bool isHostReady) { throw null; }
-#pragma warning restore SYSLIB0003
```

Comment:
This removes the API from both .NETCoreApp assemblies and the netstandard2.0 assembly. Is the .NETStandard change intentional?

Reply:
Yes, we were exploring the intentional breaking change here.  It was discussed as an option here: https://github.com/dotnet/runtime/issues/64592#issuecomment-1301475141